optimade/server/mappers/entries.py

```diff
@@ -68,12 +68,13 @@ class BaseResourceMapper:
     LENGTH_ALIASES: Tuple[Tuple[str, str]] = ()
     PROVIDER_FIELDS: Tuple[str] = ()
     ENTRY_RESOURCE_CLASS: Type[EntryResource] = EntryResource
-    RELATIONSHIP_ENTRY_TYPES: Set[str] = {"references", "structures"}
+    RELATIONSHIP_ENTRY_TYPES: Set[str] = {"references", "structures", "trajectories"}
     TOP_LEVEL_NON_ATTRIBUTES_FIELDS: Set[str] = {"id", "type", "relationships", "links"}
     SUPPORTED_PREFIXES: Set[str]
     ALL_ATTRIBUTES: Set[str]
     ENTRY_RESOURCE_ATTRIBUTES: Dict[str, Any]
     ENDPOINT: str
+    REFERENCE_STRUCTURE_FIELDS: List[str] = []
```

Comment:
I think the attribute name here is too confusing for a base class (as mentioned on Slack) as it uses the names of the two other completely independent endpoints. I would suggest making it specific to the trajectory mapper and accessing it via e.g. `hasattr` when the transformer tries to invoke it

Reply:
I have removed REFERENCE_STRUCTURE_FIELDS from the BaseResourceMapper.